apps/web/src/pages/Communities/CommunityMembersPage/CommunityMembersPage.tsx

```diff
@@ -165,24 +173,58 @@ interface RouteParams extends Record<string, string> {
 }
 
 const CommunityMembersPage = () => {
+  const intl = useIntl();
+  const paths = useRoutes();
+
   const { communityId } = useRequiredParams<RouteParams>("communityId");
   const [{ data, fetching, error }] = useQuery({
     query: CommunityMembersTeam_Query,
     variables: { communityId },
   });
 
   const community = data?.community;
+  const formattedPageTitle = intl.formatMessage(pageTitle);
+
+  const {
+    communityName,
+    navigationCrumbs: baseCrumbs,
+    navTabs,
+  } = useOutletContext<ContextType>();
+
+  const crumbs = [
+    ...(baseCrumbs ?? []),
+    {
+      label: intl.formatMessage({
+        defaultMessage: "Manage access",
+        id: "J0i4xY",
+        description: "Title for members page",
+      }),
+      url: paths.communityManageAccess(communityId),
+    },
+  ];
 
   return (
-    <AdminContentWrapper>
-      <Pending fetching={fetching} error={error}>
-        {community ? (
-          <CommunityMembers communityQuery={community} />
-        ) : (
-          <ThrowNotFound />
-        )}
-      </Pending>
-    </AdminContentWrapper>
+    <>
+      <SEO title={formattedPageTitle} />
+      <Hero
+        title={
+          fetching ? intl.formatMessage(commonMessages.loading) : communityName
+        }
+        crumbs={crumbs}
+        navTabs={navTabs}
+      />
+      <div data-h2-wrapper="base(center, large, x1) p-tablet(center, large, x2)">
```

Comment:
Worth updating this now to use full width since we know [that will come down to pipe eventually](https://github.com/GCTC-NTGC/gc-digital-talent/issues/12000)? Or keep it like this for consistency sake and update it as part of that other ticket?

Reply:
I think I'd like to leave it for now.  For one, I don't have designs and for two, it will probably be more consistent if they're all done at the same time by one person.

Reply:
Make sense :+1: